include/photo_album.h

```diff
@@ -6,12 +6,12 @@
 
 #define PHOTO_ALBUM_MAX 36
 
-typedef struct PHOTO_MON {
+typedef struct PhotoMon {
   u16 species;
   u8 form;
-  u8 shiny : 1;
-  u8 gender : 1;
-} PHOTO_MON;
+  u8 shiny:1;
+  u8 gender:1;
```

Comment:
still 2 spaces, please make these 4....
same for the entire file...

Reply:
ffs vscode fell off